setupJest.js

```diff
@@ -0,0 +1 @@
+require('jest-fetch-mock').enableMocks()
```

Comment:
again, linter would have fixed this missing newline. These are not a big deal, just a habit you need to get into.